gitman/cli.py

```diff
@@ -35,6 +35,10 @@ def main(args=None, function=None):  # pylint: disable=too-many-statements
                          help="overwrite uncommitted changes in dependencies")
     options.add_argument('-c', '--clean', action='store_true',
                          help="delete ignored files in dependencies")
+    options.add_argument('-s', '--skip-changes', action='store_true',
+                         dest='skip_changes',
+                         help="skip uncommitted changes in dependencies")
```

Comment:
I think the wording should be "skip dependencies with uncommitted changes" to make it clear that the entire dependency is skipped, not just the modified files.